src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -164,16 +164,25 @@ In the [authentication domain UI](#ui), under **Management**, you can control so
 
 ## Manage user type [#user-upgrade]
 
-In the [authentication domain UI](#ui), under **Management**, you can control how basic users upgrade to [full users](/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure). You have different options depending on whether you are managing user type in New Relic or via SCIM: 
+In the [authentication domain UI](#ui), under **Management**, you can control how your users' [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type) is managed. This includes how the user type can be edited and how basic user requests to become full users are handled. 
+
+There are two main settings: 
+* **Manage user type in New Relic**: This is the default option. It allows you to manage your users' user type from New Relic. 
```

Comment:
is "user type" the official name of this entity?  If so, should this word have a different format per public docs style guides? If not, should we use the official name in various places of the doc rather than "user type" in order to clarify?

Reply:
yeah 'user type' is the name of the thing. i think it's fine fwiw. it's like saying 'manage password settings' to me.